openstates/de/bills.py

```diff
@@ -1,693 +1,329 @@
-
-        return vote
-
-    def scrape_documents(self, source, docname, filename, tmp, session_num,
-                         re_docnum=re.compile(r'var F?docnum="(.+?)"'),
-                         re_moniker=re.compile(r'var moniker="(.+?)"'),
-                         **kwargs):
-        '''
-        Returns a generator like [{'name': 'docname', 'url': 'docurl'}, ...]
-        '''
-        source = source.replace(' ', '+')
-
-        try:
-            _doc = self._url_2_lxml(source)
-        except scrapelib.HTTPError:
-            # Grrr...there was a dead link posted. Warn and skip.
-            msg = 'Related document download failed (dead link): %r' % source
-            self.warning(msg)
-            return
-
-        if _doc.xpath('//font[contains(., "DRAFT INFORMATION")]'):
-            # This amendment is apparently still in draft form or can't
-            # be viewed for security reasons, but we can still link to
-            # its status page.
-            yield dict(name=docname, url=source)
-            return
+            if "committee" in title_text:
+                pass
+                #the committee reports let a legislator
+                #comment on a bill. They can comment as
+                #"favorable","unfavorable" or "on its merits"
+                #but these are NOT votes (per conversation w
+                #seceretary of the DE senate 3/16/15). The bill is
+                #considered if the majority sign it, which will
+                #appear in the bill's action history as being
+                #reported out of committee
+
+            if "voting" in title_text:
+                vote_info = tds[1].xpath('./a')
+                for v in vote_info:
+                    vote_name = v.text_content().strip()
+                    vote_documents[vote_name] = v.attrib["href"]
+                
+
+            if "actions history" in title_text:
+                action_list = tds[1].text_content().split("\n")
+
+
+        if sub_link:
+            bill = self.scrape_bill(sub_link,chamber,session)
+            bill.add_title(bill_id)
 
-        # The full-text urls are generated using onlick javascript and
-        # window-level vars named "moniker" and "docnum".
-        if docname == "Fiscal Note":
-            xpath = '//script[contains(., "var Fdocnum")]'
         else:
-            xpath = '//script[contains(., "var docnum")]'
-        script_text = _doc.xpath(xpath)[0].text_content()
-        docnum = re_docnum.search(script_text).group(1)
-        moniker = re_moniker.search(script_text).group(1)
-
-        # Mimetypes.
-        formats = ['.html', '.pdf', '.docx', '.Docx']
-        mimetypes = {
-            '.html': 'text/html',
-            '.pdf': 'application/pdf',
-            '.docx': 'application/msword'
-            }
-
-        for format_ in formats:
-
-            el =_doc.xpath('//font[contains(., "%s%s")]' % (filename, format_))
+            bill = Bill(session,chamber,bill_id,bill_title)
```

Comment:
fixed spacing issue, found issue with duplicate versions from substitute bill, fixed that, too.